mkosi/config.py

```diff
@@ -2251,7 +2251,16 @@ def cache_manifest(self) -> dict[str, Any]:
             "release": self.release,
             "mirror": self.mirror,
             "architecture": self.architecture,
-            "package_manager": self.distribution.package_manager(self).executable(self),
+            # Caching the package manager used does not matter for the default tools tree because we don't
+            # cache the package manager metadata for the tools tree either. In fact, it can cause issues when
+            # as the cache manifest for the tools tree will be different sometimes depending on whether we're
```

Comment:
```suggestion
            # cache the package manager metadata for the tools tree either. In fact, it can cause issues
            # as the cache manifest for the tools tree will sometimes be different depending on whether we're
```